assets/controllers/rich_textarea_controller.js

```diff
@@ -8,6 +8,8 @@ export default class extends Controller {
     }
 
     handleInput (event) {
+        let wrapKeys = ['`', '*', '_', '"', '\''];
```

Comment:
should `~` be added to this for strikethrough?

Reply:
yeah I think I could add that

Reply:
added